### ISSUE TITLE
Empty service definition in proto file produces bad go code

### ISSUE THREAD

**Issue by ofpiyush** [twitchtv/twirp#171]
Steps to reproduce:

Try compiling this
```
syntax = "proto3";
package foo;
service Bar {
}
```

Errors:
```
imported and not used: "strings"
prefix declared and not used
```

Fixes:

Add a method to prevent import failing right after import statements
```
var _ = strings.Index
```

// Add empty use to prevent prefix failing in client functions
```
_ = prefix
```

**Comment by spenczar** [comment]
Thanks for the bug report. I'll put together a fix and a release.

**Comment by spenczar** [comment]
Fixed in #172, release v5.7.0.
